Resources/datasets/market-hours/index/usa/generic/time-zone.html

```diff
@@ -1,3 +1,3 @@
 <!-- Code generated by market-hour-code-generator.py -->
 
-<p>The US Index market trades in the <code>America/Chicago</code> time zone.</p>
\ No newline at end of file
+<p>The US Index market trades in the <code>America/Chicago, America/New York</code> time zone.</p>
```

Comment:
These should be separated by an "and". If there are more than two, maybe we should use a list

> The US Index market trades in the following time zones:
> - <code>America/Chicago</code>
> - <code>America/New York</code>
> - <code>America/New York</code>

Reply:
If there are multiple time zones, "time zone" needs to be "time zones".

Reply:
How about we use a list for all cases? It looks easier to follow to me.